### ISSUE TITLE
Missing bool from Data.Bool

### ISSUE THREAD

**Issue by TazeTSchnitzel** [valderman/haste-compiler#310]
raycast.hs:6:19: Module ‘Data.Bool’ does not export ‘bool’

[Easy to implement](http://hackage.haskell.org/package/base-4.8.0.0/docs/src/Data-Bool.html#bool):

```Haskell
bool :: a -> a -> Bool -> a
bool f _ False = f
bool _ t True  = t
```

**Comment by valderman** [comment]
This is caused by Haste using a patched `base` from GHC 7.6, where `bool` wasn't implemented yet. Will be fixed when 7.10 support arrives, since all of `base` will be getting upgraded to the latest version this time around.

**Comment by TazeTSchnitzel** [comment]
Ah, I see. I didn't realise `bool` was a recent addition. :)

**Comment by valderman** [comment]
Fixed on master with GHC 7.10.
